src/views/poi_bloc/Images.jsx

```diff
@@ -0,0 +1,32 @@
+/* global _ */
+import React from 'react';
+import { toCssUrl } from 'src/libs/url_utils';
+
+const ImagesBlock = ({
+  block,
+  poi,
+}) => {
+  const images = block.images.filter(img => img.url !== poi.topImageUrl).slice(0, 3);
+  if (!images || images.length === 0) {
+    return null;
+  }
+
+  return <div className="poi_panel__pictures">
+    <h5 className="poi_panel__pictures_title">
+      <i className="icon-icon_camera"></i>
+      { _('Photos') }
+    </h5>
+    <div className='poi_panel__pictures_tiles'>
```

Comment:
Can you please keep consistent attribute quotes? You used `"` above, and `'` here and below. `"` is the preferred way, used everywhere else in the code.
And yes I know this should be an ESLint rule, I'll add it later :) And I'm only asking for this change because you have other real linting errors, so you can fix it at the same time.

Reply:
No? :crying_cat_face: